docs/Synthesis UI.md

```diff
@@ -0,0 +1,92 @@
+# UI interactions in various modes
+
+Document describes modes that browser.html could be in & how various UI components will behave in that mode based of the **Synthesis UI / Jun 15, 2015** specification.
+
+## Actions
+
+- `ShowWebView`
+- `ShowWebViewByID`
+- `CreateWebView`
+- `EditWebView`
+- `ChooseNextWebView`
```

Comment:
As with the old UI, we will also need "highlightNextWebView" and "highlightPreviousWebView" for dealing with `alt tab`.

Alt-tab lets you highlight a webview before choosing it. When you release the keys, it is chosen. This is also different from editWebView because we won't be editing the URL in `alt tab` mode.